mod/01_rating/set/right/variables.rb

```diff
@@ -87,7 +87,10 @@ def default_edit_args args
   end
 
   def metric_list
-    Card.search(type_id: MetricID, limit: 0).map do |m|
+    metrics = Card.search(type_id: MetricID,
```

Comment:
this refers to https://www.pivotaltracker.com/story/show/118517081 `make wikirating selector use only scores`. It will only shows score metrics now instead of all metrics.